bgpd/bgp_linkstate_tlv.c

```diff
@@ -29,24 +29,26 @@ struct bgp_linkstate_tlv_info {
 #define MAX_SZ 0xFFFF
 #define UNDEF_MULTPL 1
 
+/* clang-format off */
 struct bgp_linkstate_tlv_info bgp_linkstate_tlv_infos[BGP_LS_TLV_MAX] = {
```

Comment:
why clang is disabled?

Reply:
because clang gives an horrible output and checkpatch says that clang output is not correct